Podfile

```diff
@@ -3,7 +3,7 @@ source 'https://github.com/CocoaPods/Specs'
 workspace 'VideoQuickStart'
 
 abstract_target 'TwilioVideo' do
-  pod 'TwilioVideo', '2.0.0-beta4'
+  pod 'TwilioVideo', '~> 2.0.0'
```

Comment:
I figured that matching patch versions was the right thing to do. Minor updates should be a case where we want to update sample code, or consciously make the decision not to update it.